lib/completions/endpoints/base.rb

```diff
@@ -90,6 +90,7 @@ def perform_completion!(prompt, user, model_params = {})
 
                     begin
                       partial = extract_completion_from(raw_partial)
+                      next if partial.blank?
```

Comment:
you get a blank partial from a non-blank raw_partial when streaming and getting an special response, like the last message in a stream from huggingface